spec/unit/default_spec.rb

```diff
@@ -70,6 +70,39 @@
       user: 'whats_fresh')
   end
   # rubocop:enable Metrics/LineLength
+
+  it 'installs and sets up gunicorn for Whats Fresh' do
+    expect(chef_run)
+      .to create_gunicorn_config('/opt/whats_fresh/gunicorn_config.py').with(
+        listen: '0.0.0.0:8888')
+  end
+
+  it 'installs and sets up supervisor for Whats Fresh' do
+    expect(chef_run)
+      .to enable_supervisor_service('whats_fresh').with(
+        command: '/opt/venv_whats_fresh/bin/gunicorn' \
+          ' whats_fresh.wsgi:application' \
+          ' -c /opt/whats_fresh/gunicorn_config.py',
+        autorestart: true,
+        directory: '/opt/whats_fresh')
+  end
+
+  it 'installs gunicorn to Whats Fresh virtualenv' do
```

Comment:
It sounds like the cookbook requires a webapp for testing.

I would consider building the "hello world" Django app from the tutorial: https://docs.djangoproject.com/en/1.8/intro/tutorial01/  This has the advantage of being a) real, b) familiar, and c) very likely to be updated and maintained by the Django folks.  The only thing better would be an official version we could reference here on github, but those folks don't host a copy.  You could then name it 'tutorial-a', and use it for actual integration testing.  What do you think?